[CS] Code Style fixes for libraries Object operator not indented correctly

photodude opened the pull request:
Pull Request for Issue Object operator not indented correctly.

### Summary of Changes
- autofixed cases of Object operator not indented correctly

[Automatically fixed with Joomla code standards 2.0.0 PHPCS2-alpha2 fixers](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2)

None of the manual only fixes have been applied

### Testing Instructions
Merge by code review

### Expected result
code style has been applied as listed above, old code style testing on drone does not error.

### Actual result
code style had not been applied. [Autofixers from the Joomla code standards 2.0.0 PHPCS2 alpha2](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2) were used to implement fixable code style

### Documentation Changes Required
none

photodude committed:
Object operator not indented correctly

photodude committed:
Tabs must be used to indent lines; spaces are not allowed

Rerun the autofixer to fix space indenting

Quy commented on libraries/src/Updater/UpdateAdapter.php:
```diff
@@ -183,9 +183,9 @@ protected function getUpdateSiteName($updateSiteId)
-					->where($db->qn('update_site_id') . ' = ' . $db->q($updateSiteId));
+			->select($db->qn('name'))
+			->from($db->qn('#__update_sites'))
+			->where($db->qn('update_site_id') . ' = ' . $db->q($updateSiteId));
```
Insert blank line below.

photodude replied:
You want a line between `$query = $db->` and `$db->setQuery($query);` ?

Quy commented on libraries/src/Installer/Adapter/ComponentAdapter.php:
```diff
@@ -803,8 +803,8 @@ public function uninstall($id)
-					->delete('#__schemas')
-					->where('extension_id = ' . $id);
+			->delete('#__schemas')
+			->where('extension_id = ' . $id);
```
Insert blank line below.

photodude replied:
you want a line between `$query = $db->` and `$db->setQuery($query);`?

Quy replied:
Yes, elsewhere it it like that.

photodude replied:
There is nothing in the code style that requires an additional line here. There is also nothing in the code style that forbids having the extra line. I wrote the example in the code style PHP SQL section and the only reason there are extra lines in the example is due to the comments. 

There are places throughout the repo (even throughout this file) showing with and without the extra line. Unless we include a requirement on this in the code style, I'm of the opinion that either way should be acceptable.

Quy replied:
I don't know for sure. I am just going by the examples here:
https://docs.joomla.org/Selecting_data_using_JDatabase

photodude replied:
So in that example it's the comments that require the extra line. There is one in that group that breaks that pattern but in my opinion there is no hard or fast rule beyond the new line before a comment .

Quy commented on libraries/src/Installer/Adapter/ComponentAdapter.php:
```diff
@@ -1176,10 +1176,10 @@ protected function _updateMenus($component_id, $clientId = null)
+			->update('#__menu')
+			->set('component_id = ' . $db->quote($component_id))
+			->where('type = ' . $db->quote('component'))
+			->where('(' .
```
Fix indentation below.

photodude replied:
fixed, it's obvious with concatenation on the left-hand side.

photodude committed:
fix indentation

- Table names and table column names should always be enclosed in the
quoteName() method to escape the table name and table columns. Field
values checked in a query should always be enclosed in the quote()
method to escape the value before passing it to the database.

photodude committed:
$db->quoteName is not used here, these are values

Quy commented on libraries/src/Installer/Adapter/ComponentAdapter.php:
```diff
@@ -1176,13 +1176,13 @@ protected function _updateMenus($component_id, $clientId = null)
+			->set('component_id = ' . $db->quote($component_id))
+			->where('type = ' . $db->quote('component'))
+			->where('('
+					. 'link LIKE ' . $db->quote('index.php?option=' . $option) . ' OR '
```
Reduce indentation.

photodude replied:
reduced by one tab

photodude committed:
Reduce indentation
